### PR TITLE
feat: client inactivity callback

### PR DESCRIPTION
**Pull request by lukasberbuer**
Add new member function to set inactivity callback: `Client::onInactive`

Closes #284

### CODE REVIEW

File: src/Client.cpp [hunk 1]
```diff
@@ -324,6 +324,17 @@ void Client::onSessionClosed(StateCallback callback) {
     setStateCallback(*this, detail::ClientState::SessionClosed, std::move(callback));
 }
 
+void Client::onInactive(InactivityCallback callback) {
+    detail::getContext(*this).inactivityCallback = std::move(callback);
+    detail::getConfig(*this).inactivityCallback = [](UA_Client* client) noexcept {
+        log(client, LogLevel::Info, LogCategory::Client, "lalala");
```
Comment (ErikTempelaarVO) [inline src/Client.cpp:330]: This must be for debugging purposes

**Review by ErikTempelaarVO — APPROVED** [review]

File: src/Client.cpp (hunk 1 above)
Comment (lukasberbuer) [reply src/Client.cpp]: Haha, indeed 😅

**Comment by lukasberbuer** [comment]
@ErikTempelaarVO, did you verify if this fixes #284?

**Comment by ErikTempelaarVO** [comment]
I only reviewed the code. Have not tried it yet. Unable to do so now. Maybe later this week.

**Pull request merged**